src/components/pages/champions/PatchNotes.tsx

```diff
@@ -48,7 +48,10 @@ function PatchInfo({ patch }: PatchInfoProps) {
   const { enName, version, target, targetImgUrl, changes } = patch;
   const capitalizedEnName = enName.charAt(0).toUpperCase() + enName.slice(1);
   // 기본 능력치의 경우 스킬 이미지가 없기 때문에 챔피언 초상화를 사용
-  const imgUrl = targetImgUrl !== null ? `https${targetImgUrl.split('f=http')[1]}` : championIconUrl(capitalizedEnName);
+  const imgUrl =
+    targetImgUrl !== null
+      ? targetImgUrl.split('f=')[1].replace('http://', 'https://')
+      : championIconUrl(capitalizedEnName);
```

Comment:
```suggestion
  const imgUrl =
    targetImgUrl !== null
      ? new URL(targetImgUrl).searchParams.get('f').replace(/^http:\/\//, 'https://')
      : championIconUrl(capitalizedEnName);
```
혹시 모를 예외케이스 때문에 이쪽이 더 나을 것 같은데 어떻게 생각하시나요?